### PR TITLE
uploading assignment4 and assignment5

### PR DESCRIPTION
**Pull request by poonam-khode**

### CODE REVIEW

File: poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Assignment 5 Bootstrap</title>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+    </head>
+    <body>
+        <div class="container mt-5">
+            <div class="row">
+                <div class="col">
+                    <ul class="nav">
```
Comment (MKadeer17) [inline poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html]:
```suggestion
                    <ul class="nav">
```
Remove this Full <ul class="nav"></ul>
Use Breadcrumb not nav
Like this: **Add this**
```
<nav class="my-5 ms-3" aria-label="breadcrumb">
            <ol class="breadcrumb">
              <li class="breadcrumb-item"><a href="#">Home</a></li>
              <li class="breadcrumb-item"><a href="#">Employee</a></li>
              <li class="breadcrumb-item active" aria-current="page">Employee Details</li>
            </ol>
        </nav>
```

File: poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                        </li>
+                      </ul>
+                </div>
+            </div>
+            <div class="row mt-5">
+                <div class="col">
+                    <h3>List of Employees <span class="badge badge-info">New</span></h3>
+                    <table class="table">
+                        <thead class="bg-light">
+                          <tr>
```
Comment (MKadeer17) [inline poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html:28]:
```suggestion
                          <tr>
```
After this `<tr>` add one more column
`<th>SrNo</th>`

File: poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                            <th>Firstname</th>
+                            <th>Lastname</th>
+                            <th>Email</th>
```
Comment (MKadeer17) [inline poonam/bootstrap/Bootstrap_assignment5/Assign_5bootstrap.html]:
```suggestion
                            <th>Email</th>
```
Not Email
It's Mobile No
```suggestion
+<th>Mobile No</th>
```

**Review by MKadeer17 — APPROVED** [review]
Only Little Bit Changes

**Review by MKadeer17 — APPROVED** [review]
Good

**Comment by poonam-khode** [comment]
> Good

Thank u quadir

**Review by sanketmod — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**